tasks/provision_service.rb

```diff
@@ -19,9 +19,9 @@ def default_uri
   def platform_to_cloud_request_parameters(platform, cloud, region, zone)
     case platform
     when String
-      { cloud: cloud, region: region, zone: zone, images: [platform] }
+      { cloud:, region:, zone:, images: [platform] }
     when Array
-      { cloud: cloud, region: region, zone: zone, images: platform }
+      { cloud:, region:, zone:, images: platform }
```

Comment:
is this new syntax yeah

Reply:
yep, thats Ruby's 3.1+ recommended syntax. Rubocop safely autocorrected it